contracts/proposal/dao-proposal-single-instant/src/testing/test_tube.rs

```diff
@@ -520,6 +549,185 @@ pub mod test_tube {
         );
     }
 
+    #[test]
+    #[ignore]
+    /// Test case of a proposal failing due to be replayed
+    fn test_dao_proposal_single_instant_ko_replay() {
```

Comment:
nit: The exact logic of what this test tests and how it asserts that a replay happens is slightly hard to follow

Reply:
i removed unnecessary assertions around admin and treasury balances, as they are out of context and already tested out on the bank_send_ok test case. I also enhanced the comments around what the test is doing on the relevant lines. Let me know if thats fine!